gpgme.gemspec

```diff
@@ -17,7 +17,7 @@ Made Easy). GnuPG Made Easy (GPGME) is a library designed to make access to
 GnuPG easier for applications. It provides a High-Level Crypto API for
 encryption, decryption, signing, signature verification and key management.}
 
-  s.add_runtime_dependency "mini_portile2", "~>2.7.0"
+  s.add_runtime_dependency "mini_portile2", ">= 2.7.0",  "< 3"
```

Comment:
What's wrong with `~> 2.7`? That would express the same with little less boolean logic